jest.config.js

```diff
@@ -1,31 +1,31 @@
 /* eslint-disable @typescript-eslint/no-var-requires */
-const { jestConfig } = require("@activeviam/activeui-sdk-scripts");
-const pkgDir = require("pkg-dir");
-
-const root = pkgDir.sync();
-
-if (!root) {
-  throw new Error("Could not find root.");
-}
+const isCI = require("is-ci");
 
 const extensions = ["js", "jsx", "ts", "tsx", "mjs", "json"];
-const esmPackages = [
-  "@activeviam/utils",
-  "monaco-editor",
-  "lodash-es",
-  "@react-dnd",
-  "react-dnd",
-  "dnd-core",
-];
+const esmPackages = ["@activeviam/*", "lodash-es", "monaco-editor"];
 
-module.exports = {
-  ...jestConfig,
-  reporters: ["default"],
+// Note that in addition to the config below, there are several mocked modules under the __mocks__ folder.
+// They correspond to modules that are UI/browser-related and are transitively pulled by @activeviam dependencies, but are not necessary for `activeui-migration` to work in a Node environment.
+const config = {
```

Comment:
Can we use the config exported by `@activeviam/jest-config-activeui` package ?

Reply:
No as this package is private.